### PR TITLE
Bug/br fix redis port

### PR DESCRIPTION
**Pull request by briri**
Fix for #180 

As part of getting the Cache working again on stage, I switched us from serverless Redis to a ValKey cluster.

### CODE REVIEW

**Comment by briri** [comment]
looks like this includes some things that merged into main but have not yet been back-filled into the development branch

File: templates/cache-backend.yaml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  #     Description: !Sub "${AWS::StackName} SecGrp"
+
+  CacheSecurityGroup:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      GroupDescription: !Sub "${AWS::StackName} SecGrp"
+      GroupName: !Sub '${AWS::StackName}-cache-sec'
+      VpcId: !Ref VpcId
+
+  # CacheSecurityGroupIngress:
```
Comment (jupiter007) [inline templates/cache-backend.yaml:78]: L66 & L78 Is the commented out code left here for a reason?

**Review by jupiter007 — APPROVED** [review]
I only had one question. Otherwise, it looks good to me.

File: templates/cache-backend.yaml (hunk 1 above)
Comment (briri) [reply templates/cache-backend.yaml:78]: nope we ended up not needing it. I will remove

**Pull request merged**